DOCS/man/vo.rst

```diff
@@ -881,6 +881,15 @@ Available video output drivers are:
 
         Windows only.
 
+    ``dcomposition=<yes|no>``
+        Allows DirectComposition when using the ANGLE backend (default: yes).
+        Direct Composition is a Winows 8+ feature which improves rendering
```

Comment:
Maybe something like this:

DirectComposition implies flip-model presentation, which can improve rendering efficiency on Windows 8+ by avoiding a copy of the video frame. mpv uses it by default where possible, but it can cause poor behaviour with some drivers, such as a black screen or graphical corruption when leaving full-screen mode.